src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -977,6 +977,46 @@ int publish_abort(reservation_t& res) {
   return 0;
 }
 
+int get_persistent_queue_by_topic_name(const DoutPrefixProvider* dpp, librados::IoCtx& rados_ioctx, const std::string& topic_name,
+                                       std::size_t& number_of_reservations, uint32_t& number_of_entries, uint64_t& queue_size, optional_yield y)
+{
+  librados::ObjectReadOperation op;
+  bufferlist bl;
+  int rval = 0;
+  op.read(0, 0, &bl, &rval);
+  if(rval < 0) {
+    ldpp_dout(dpp, 1) << "ERROR: something failed in preparing read operation: " << rval << dendl;
+    return rval;
+  }
+
+  auto ret = rgw_rados_operate(dpp, rados_ioctx, topic_name, &op, &bl, y);
```

Comment:
why do you need this call?
do you use it later on?

Reply:
no